project/app/api/dbpgsql.py

```diff
@@ -29,51 +32,36 @@ def __init__(self):
-               'form_form_name',
-               'form_created_by',
-               'proposed_bridge_location_gps_latitude',
-               'proposed_bridge_location_gps_longitude',
-               'current_crossing_method',
-               'nearest_all_weather_crossing_point',
-               'days_per_year_river_is_flooded',
-               'flood_duration_during_rainy_season',
-               'market_access_blocked_by_river',
-               'education_access_blocked_by_river',
-               'health_access_blocked_by_river',
-               'other_access_blocked_by_river',
-               'primary_occupations',
-               'primary_crops_grown',
-               'river_crossing_deaths_in_last_3_years',
-               'river_crossing_injuries_in_last_3_years',
-               'incident_descriptions',
-               'notes_on_social_information',
-               'cell_service_quality',
-               'four_wd _accessibility',
-               'name_of_nearest_city',
-               'name_of_nearest_paved_or_sealed_road',
-               'bridge_classification',
-               'flag_for_rejection',
-               'rejection_reason',
-               'bridge_type',
-               'estimated_span_m',
-               'height_differential_between_banks',
-               'bridge_opportunity_general_project_photos',
-               'bridge_opportunity_casesafeid',
-               'senior_engineering_review_conducted',
-               'country']
-
+    columns1 = ['bridge_name', 'bridge_opportunity_project_code',
+       'bridge_opportunity_needs_assessment',
+       'bridge_opportunity_level1_government',
+       'bridge_opportunity_level2_government',
+       'bridge_opportunity_gps_latitude', 'bridge_opportunity_gps_longitude',
+       'bridge_opportunity_bridge_type', 'bridge_opportunity_span_m',
+       'bridge_opportunity_individuals_directly_served',
+       'bridge_opportunity_comments', 'form_form_name', 'form_created_by',
+       'proposed_bridge_location_gps_latitude',
+       'proposed_bridge_location_gps_longitude', 'current_crossing_method',
+       'nearest_all_weather_crossing_point', 'days_per_year_river_is_flooded',
+       'flood_duration_during_rainy_season', 'market_access_blocked_by_river',
+       'education_access_blocked_by_river', 'health_access_blocked_by_river',
+       'other_access_blocked_by_river', 'primary_occupations',
+       'primary_crops_grown', 'river_crossing_deaths_in_last_3_years',
+       'river_crossing_injuries_in_last_3_years', 'incident_descriptions',
+       'notes_on_social_information', 'cell_service_quality',
+       'four_wd _accessibility', 'name_of_nearest_city',
+       'name_of_nearest_paved_or_sealed_road', 'bridge_classification',
+       'flag_for_rejection', 'rejection_reason', 'bridge_type',
+       'estimated_span_m', 'height_differential_between_banks',
+       'bridge_opportunity_general_project_photos',
+       'bridge_opportunity_casesafeid', 'country', 'good_site']
+
```

Comment:
This second set is for the prediction table correct?

Reply:
yes, one table is cleaned main dataset. the second one is to get the query features for prediction purpose

Reply:
Super! All Approved on my end.

Reply:
The model is housed in b2p-d.ipynb notebook, named gs_model.best_estimator_ and serialized as gs_model.

Reply:
Sounds good